src/structures/stack.js

```diff
@@ -0,0 +1,22 @@
+class myStack {
+  constructor() {
+    this.elements = [];
```

Comment:
For a static property, we don't need to declare it in the constructor. Refer to: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields